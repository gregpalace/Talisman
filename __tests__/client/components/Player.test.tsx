import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import Player from '../../../src/client/components/player';

describe('React unit tests', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<Player />).dive();
  });

  describe('<Player />', () => {
    it('Should render correctly', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

});