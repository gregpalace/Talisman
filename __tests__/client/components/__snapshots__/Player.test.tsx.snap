// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React unit tests <Player /> Should render correctly 1`] = `undefined`;
